.buildkite/macos/macos.rayci.yml

```diff
@@ -102,7 +102,7 @@ steps:
     commands:
       - RAY_INSTALL_JAVA=1 ./ci/ray_ci/macos/macos_ci.sh run_ray_cpp_and_java
 
-  - label: ":ray: core: :mac: small & large flaky tests"
+  - label: ":ray: core: :mac: small & medium & large flaky tests"
```

Comment:
maybe just remove the size? are there sizes other than small/medium/large?

Reply:
sound good let me update